h2o-py/tests/testdir_algos/automl/leaderboard/pyunit_automl_leaderboard_binary.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function
+import os
+import sys
+
+sys.path.insert(1, os.path.join("..","..","..",".."))
+from h2o.automl import H2OAutoML
+from tests import pyunit_utils as pu
+
+from _leaderboard_utils import check_leaderboard, check_model_property
+from _automl_utils import import_dataset, get_partitioned_model_names
```

Comment:
@michalkurka this is as much magic as I can sacrifice to, you pushed my boundaries :)
it's not very pythonic (the provenance of `_automl_utils` in the parent folder is not completely obvious here, although not difficult to find in this case), but I think it is clear enough.

if 2 utility modules with the same name are created, it will fail immediately with a descriptive message, encouraging people to give a more precise name to the module (instead of just `_utils.py`)